ci-operator/step-registry/cucushift/installer/rehearse/azure/ipi/ovn/hypershift/provision/cucushift-installer-rehearse-azure-ipi-ovn-hypershift-provision-chain.yaml

```diff
@@ -0,0 +1,13 @@
+chain:
+  as: cucushift-installer-rehearse-azure-ipi-ovn-hypershift-provision
+  steps:
+  - chain: cucushift-installer-rehearse-azure-ipi-ovn-provision
+  - chain: hypershift-install
+  - chain: hypershift-azure-create
+  env:
+  - name: HYPERSHIFT_BASE_DOMAIN
+    default: "qe.azure.devcluster.openshift.com"
+  - name: HYPERSHIFT_AWS_REGION
+    default: "us-east-2"
```

Comment:
region is incorrect

Reply:
https://github.com/openshift/release/pull/33460 wait for merge